crates/miden-rpc-proto/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use std::{fs::{self, File}, io::Write, path::Path};
+
+mod proto_files;
+pub use proto_files::PROTO_FILES;
+
+/// Writes the RPC protobuf file into `target_dir`.
+pub fn write_proto(target_dir: &Path) -> Result<(), String> {
+    if !target_dir.exists() {
+        fs::create_dir_all(target_dir).unwrap();
```

Comment:
Should we return an error here explaning that we failed to create the directory? Same goes for some `unwraps()` later in this file.

Reply:
Yeah, was planning on removing these. Done!